lib/voyager_helpers/liberator.rb

```diff
@@ -796,11 +796,20 @@ def merge_holding_item_into_bib(bib, holding, item, recap=false, conn=nil)
             h[key]['subfields'].insert(2, {"h"=>combined_call_no})
             record_hash['fields'] << h
           end
+          item_enum_chron = ''
+          if item[:enum]
+            item_enum_chron << "#{item[:enum]} "
+          end
+          if item[:chron]
+            item_enum_chron << "(#{item[:chron]})"
+          end
+          item_enum_chron.gsub!(/^(.*)[ ]+$/, '\1')
```

Comment:
I don't think we've addressed these comments yet @tampakis. We should wrap this one up next week.